source/documentors/how-tos/create_github_issue.rst

```diff
@@ -0,0 +1,81 @@
+How to Create a GitHub Issue
+#################################
+
+This how-to will help you create a GitHub issue related to the Open edX documentation.
+
+.. contents:: Contents
+  :local:
+  :depth: 1
+
+Prerequisites
+*************
+
+* You have access to the Open edX documentation repository: `docs.openedx.org <https://github.com/openedx/docs.openedx.org>`_.
```

Comment:
This isn't a specific prerequisite. Everyone has access to the repository. The only prereqs are a GitHub account.